admin/admin_settings.php

```diff
@@ -100,10 +101,13 @@
 	}
 	update_settings_value("nyhed", "dashboard", $newssnippet, "The news snippet showen to all admin accounts on this system");
 	db_modify($qtxt,__FILE__ . " linje " . __LINE__);
-	$qtxt="update settings set var_value='$languageId' where var_name='languageId'";
+	$qtxt="update settings set var_value='$languageId' where var_name='language_id'";
```

Comment:
What is the logic behind changing languageId to language_id?

Reply:
The logic being that `language_id` is the value set in the table field `var_name` :-)
See here:
https://github.com/DANOSOFT/saldi/blob/377e1dc8733ed9ec06c7fc96123890f3681a831d/index/install.php#L272-L273